charts/grafana/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: grafana
-version: 6.57.4
-appVersion: 9.5.5
+version: 7.0.0
+appVersion: 10.0.1
```

Comment:
@zzzinho 
Thanks for the contribution. Can you specify what break change that the helm chart user should know, such as values.yaml change, upgrading / rollback note or strategy in README.md?

Reply:
@zzzinho , please use 6.x version for chart. There's no breaking changes, no need for "fancy" version number:

> A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.